src/ActionBar/_ActionBarBack.test.js

```diff
@@ -0,0 +1,23 @@
+import ActionBar from './ActionBar';
+import { mount } from 'enzyme';
+import React from 'react';
+
+describe('ActionBarBack Prop spreading', () => {
```

Comment:
Should just reference `ActionBarBack` (the prop spreading is covered by the actual `test`).